src/parse_grammar.rs

```diff
@@ -1,16 +1,21 @@
 // HACK(eddyb) silence warnings from unused exports in the generated code.
 #![allow(unused)]
 
+// HACK(eddyb) needed for bootstrapping `parse_grammar`.
```

Comment:
Can remove `parse_grammar` at the end since we are in the `parse_grammar` module :P.